backend/api/postman_collection/models.py

```diff
@@ -0,0 +1,136 @@
+from dataclasses import asdict, dataclass, field
+from typing import Any, Optional
+from urllib.parse import urlencode, urljoin
+
+from api.constants import ApiExecution
+from api.models import APIDeployment
+from api.postman_collection.constants import CollectionKey
+from django.conf import settings
+from utils.request import HTTPMethod
+
+
+@dataclass
+class HeaderItem:
+    key: str
+    value: str
+
+
+@dataclass
+class FormDataItem:
+    key: str
+    type: str
+    src: Optional[str] = None
+    value: Optional[str] = None
+
+    def __post_init__(self) -> None:
+        if self.type == "file":
+            if self.src is None:
+                raise ValueError("src must be provided for type 'file'")
+        elif self.type == "text":
+            if self.value is None:
+                raise ValueError("value must be provided for type 'text'")
+        else:
+            raise ValueError(f"Unsupported type for form data: {self.type}")
+
+
+@dataclass
+class BodyItem:
+    formdata: list[FormDataItem]
+    mode: str = "formdata"
+
+
+@dataclass
+class RequestItem:
+    method: HTTPMethod
+    url: str
+    header: list[HeaderItem]
+    body: Optional[BodyItem] = None
+
+
+@dataclass
+class PostmanItem:
+    name: str
+    request: RequestItem
+
+
+@dataclass
+class PostmanInfo:
+    name: str = "Unstract's API deployment"
+    schema: str = (
+        "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"  # noqa: E501
```

Comment:
What is this version v2.1.0? If it is avariable or want;s to change later, Make it a constant will be better,

Reply:
This refers to the postman collection version that we're following. I'd rather make the entire string a constant